mula/tests/integration/test_api.py

```diff
@@ -837,6 +837,17 @@ def test_list_schedules(self):
         self.assertEqual(2, response.json()["count"])
         self.assertEqual(2, len(response.json()["results"]))
 
+    def test_list_schedules_scheduler_id(self):
+        response = self.client.get(f"/schedules?scheduler_id={self.scheduler.scheduler_id}")
+        self.assertEqual(200, response.status_code)
+        self.assertEqual(2, response.json()["count"])
+        self.assertEqual(2, len(response.json()["results"]))
+
+        response = self.client.get(f"/schedules?scheduler_id={uuid.uuid4()}")
+        self.assertEqual(200, response.status_code)
+        self.assertEqual(0, response.json()["count"])
+        self.assertEqual(0, len(response.json()["results"]))
```

Comment:
Maybe for a next cleanup (since it also applies to some of the other tests): try to extract `response.json()` to a variable, such that it can be reused (and avoids recalling `json()`) and it makes debugging a bit easier, because you can watch that variable using the debugger

Reply:
You're right, makes sense. Would be a good idea to tidy up test cases a bit again.